index.html

```diff
@@ -0,0 +1,148 @@
+</head>
+
+<body>
+    <header class="nav">
+        <h1 class="logo">Cimlet</h1>
+        <ul class="list">
+            <li>
+                <a class="links" href="#">Home</a>
+            </li>
+            <li>
+                <a class="links" href="#">About</a>
+            </li>
+            <li>
+                <a class="links" href="../HTML/index.html">Products</a>
+            </li>
+            <li>
+                <a class="links" href="../html-forms/contact.html">Contact</a>
+            </li>
+        </ul>
+    </header>
+    <!-- <section class="description">
+        <div>
+            <h1>Clean & Classy</h1>
+            <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Doloremque amet quia iste itaque unde excepturi, veritatis temporibus voluptatem est non pariatur a sed incidunt optio obcaecati maxime nostrum commodi atque?</p>
+            <button class="explore-btn">Explore </button>
+        </div>
+
+    </section> -->
+    <section class="product-page">
+        <h1>Products</h1>
+        <div class="products">
+            <img src="./images/omen.jpg" alt="">
+            <div class="omen-descri">
+                <h3>HP Omen 15</h3>
+                <h4>Brand: HP</h4>
+                <label class="qty" for="qty">Quantity:</h4> <input type="text"><br>
+                <label class="price" for="price">Price:#550,000</label> <br>
+                <button class="buy-button btn">Buy now</button>
+            </div>
+        </div>
+        <div class="descript">
+            <h3>Description</h3>
+            <p>Core i-7, 10th gen  512gb SSD, 1tb HDD, 8gb RAM, 6gb Nvidia RTX 2070</p>
+        </div>
+    </section>
+    <section class=" trends ">
+        <h1>Trending Products</h1>
+        <div>
+            <div class="item ">
+                <img class="img " src="./images/spectre.jpg" alt="HPspectre ">
+                <p>Hp Spectre 13- X360 - <b>#700,000</b> </p>
+                <div>
+                    <h3>Specifications</h3>
+                    <ul>
+                        <li>Intel Core i-7, 8th gen</li>
+                        <li>Hard Drive: 256gb SSD</li>
+                        <li>Storage: 8gb RAM</li>
+                        <li>Size: 13.3"</li>
+                        <li>Color: Silver</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="item">
+
+                <div class="item">
+                    <img class="img" src="./images/samsung.jpg" alt="SamsungFlex">
+                    <p>Samsung Galaxy Book- <b>#650,000</b> </p>
+                    <h3>Specifications</h3>
+                    <ul>
+                        <li>Intel Core i-7, 10th gen</li>
+                        <li>Hard Drive:512gb SSD</li>
+                        <li>Storage: 16gb RAM</li>
+                        <li>Size: 14"</li>
+                        <li>Color: Blue</li>
+                    </ul>
+                </div>
+            </div>
```

Comment:
Format this block of code properly. I think you need to also remove one of those two div elements.